browser/main/Detail/TagSelect.styl

```diff
@@ -3,15 +3,12 @@
   align-items center
   user-select none
   vertical-align middle
-  width 100%
+  width 96%
   overflow-x scroll
```

Comment:
Can you change `overflow-x scroll` to `overflow-x auto` so the scroll bar is shown only when needed 😉